Hunting Queries/Microsoft 365 Defender/General queries/Endpoint Linux AV Signature and Platform Versions

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Please have an id field here to start with.
Id is just a standard GUID. You can generate from just about any development tool, online GUID generator, or from PowerShell via the New-GUID cmdlet.